source/how-we-work/version-control/support-branches.html.md

```diff
@@ -0,0 +1,131 @@
+
+Support branches should only be used to fix bugs, not to introduce new features.
+
+![Graph showing the main and support branch running in parallel. The support/4.6.x branch diverges from the v4.6.0 on the main branch. After the point the two branches diverge, the main branch has unreleased changes and the support branch has a tag for v4.6.1](/images/support-branch-hotfix.svg)
+
+<!-- 
+gitGraph
+    commit
+    commit tag: "v4.6.0"
+    branch "support/4.6.x"
+    checkout "main"
+    commit
+    commit
+    commit
+    checkout "support/4.6.x"
+    commit
+    commit tag: "v4.6.1"
+-->
+
+The support branch should be named using the major and minor version number of the previous release. For example, if the last release was v4.6.0 and we expect the unreleased changes on the main branch to go out as part of v4.7.0, the support branch should be named `support/4.6.x`.
+
+Make sure that the bug is also fixed on the `main` branch, otherwise the bug will be introduced in the next release. This may mean you need to raise a second pull request with the same changes against `main`.
+
+## Creating a support branch
+
+If the support branch doesn't already exist, create it by following these steps.
+
+1. Make sure you have all tags in our local version of the repo by running `git fetch --all --tags --prune`
+
+2. Run `git checkout tags/v<LAST RELEASED VERSION NUMBER> -b <NAME OF SUPPORT BRANCH>`. For example, `git checkout tags/v4.6.0 -b support/4.x`
+
+3. Push the support branch to GitHub. You should push the branch without making any further changes.
+
+4. Set up [branch protection rules] for the support branch.
+
+    When using support branches for hotfixes, the branch protection rules should be the same as the branch protection rules for the `main` branch.
+
+    When using support branches for previous major versions, the required status checks should be configured to match the status checks that run on the branch.
+
+## Making changes to a support branch
+
+Before making changes:
+
+1. Check out the support branch
+
+2. Run `nvm use` to make sure you're using the right version of Node.js and npm, then run `npm install`
+
+3. Given there may be significant differences between the development environment in main and the support branch, run the tests to make sure everything works as expected before making changes
+
+4. Create a new branch based on the support branch which you'll use to raise the pull request
+
+If you are releasing a fix for a bug that has already been fixed on the main branch, you can then cherry-pick the commits that fix the bug onto your new branch.
+
+Otherwise, make whatever changes are required and then commit your code changes, before raising a pull request with the support branch as the target branch
+
+Remember that you might also need to make the same change to the main branch or other support branches.
+
+## Releasing from a support branch
+
+Follow the documentation from the support branch to publish a release.
```

Comment:
Do we need to say something here about using the correct release docs depending on the major version? We got tripped up by this when releasing 4.7 because the docs for releasing had changed in response to the architecture changes for 5.0 but now no longer accurately reflected the release process for 4.x. It's unlikely to happen, so I'm interested in your take on if we should note this.

If we're _not_ noting this then I think we should link to the latest release docs. I'm presuming we eventually want them to live here so not something we need to deal with now necessarily.

Reply:
Isn't that what this says already?

> Follow the documentation **from the support branch** to publish a release.

Reply:
Ahh you do, I missed that on first read. I'd personally appreciate it if we spelled it out a tiny bit more:
```suggestion
Follow the documentation from the support branch to publish a release. Remember that release processes may change between major releases so follow the release process for the major version you are releasing to.
```

Reply:
Tweaked again:

> The release process may be different from the release process used on the main branch, especially when releasing from a support branch for a previous major release. You should follow the documentation from the support branch to publish a release.

Does that spell it out enough?